packages/core/jsx-runtime/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@theme-ui/core-jsx-runtime",
```

Comment:
I also don't know how npm determines which files end up in the published package. I think we need to specify the directories `jsx-runtime` and `jsx-dev-runtime` explicitly to be included.

Reply:
`files` and `npmignore`?

Reply:
[The npm docs about the `files` field](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files) say

> Omitting the field will make it default to ["*"], which means it will include all files.

We aren't using it in Theme UI so I think we don't need to change anything.